agent/engine/docker_task_engine.go

```diff
@@ -2946,3 +2954,66 @@ func (engine *DockerTaskEngine) getDockerID(task *apitask.Task, container *apico
 	}
 	return dockerContainer.DockerID, nil
 }
+
+// Sets CloudWatch Logs dual stack endpoint as awslogs-endpoint option in logging config.
+// This is needed because awslogs driver that we consume from Docker does not support
+// dual stack endpoints.
```

Comment:
nit: `does not support dual stack endpoints.` - I am understanding that the driver is not able to infer the dual stack endpoint "on its own" or "automatically". But it does support dual stack endpoints when passed using the "awslogs-endpoint" option. Does it need to be reworded?

Reply:
Your understanding is correct. I will rewrord this.

Reply:
Updated